mod/research/lib/javascript/script_research.js.coffee

```diff
@@ -18,13 +17,19 @@ decko.slotReady (slot) ->
 $(document).ready ->
   $("#main:has(>#Research_Page.slot_machine-view)").addClass("pl-0 pr-0")
 
+  $($('ul.pointer-radio-list')[0]).find('input[type="radio"]').on "click", () ->
```

Comment:
these selectors are very general and will apply to every pointer that Decko makes.  We need specific selectors that only apply to the research page.

Reply:
#Research_Page ul.pointer-radio-list ?